### PR TITLE
expo push notifications guide and example. 

### PR DESCRIPTION
**Pull request by thorwebdev**
- expo push notifications edge functions guide: https://docs-git-thor-expo-push-notifications-supabase.vercel.app/docs/guides/functions/examples/push-notifications
- expo push notifications example app

### CODE REVIEW

File: examples/user-management/expo-push-notifications/supabase/config.toml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# Extra schemas to add to the search_path of every request. public is always included.
+extra_search_path = ["public", "extensions"]
+# The maximum number of rows returns from a view, table, or stored procedure. Limits payload size
+# for accidental or malicious requests.
+max_rows = 1000
+
+[db]
+# Port to use for the local database URL.
+port = 54322
+# Port used by db diff command to initialise the shadow database.
```
[the remark this answers is not in the document]
Comment (thorwebdev) [reply examples/user-management/expo-push-notifications/supabase/config.toml:21]: This is part of Supabase CLI

File: examples/user-management/expo-push-notifications/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+The database webhook handler to send push notifications is located in [supabase/functions/push/index.ts](./supabase/functions/push/index.ts). Deploy the function to your linked project and set the `EXPO_ACCESS_TOKEN` secret.
+
+1. `supabase functions deploy push`
+1. `supabase secrets set --env-file .env.local`
+
+## Create the database webhook
+
+Navigate to the [Database Webhooks settings](https://supabase.com/dashboard/project/_/database/hooks) in your Supabase Dashboard.
+
+1. Enable and create a new hook.
```
Comment (charislam) [inline examples/user-management/expo-push-notifications/README.md:43]: same feedback as in the docs site file

**Review by charislam — APPROVED** [review]

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 3]
```diff
@@ -899,6 +903,10 @@ export const functions: NavMenuConstant = {
       url: undefined,
       items: [
         { name: 'Dart Edge on Supabase', url: '/guides/functions/dart-edge' },
+        {
+          name: 'Expo Push Notifications',
+          url: '/guides/functions/examples/push-notifications',
+        },
```
Comment (dshukertjr) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:909]: Would it be better to just have a single menu item for push notifications? We could probably leave the one at the top?

Comment (thorwebdev) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:909]:
```suggestion
```

File: apps/docs/pages/guides/functions/examples/push-notifications.mdx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+```
+
+## Create the database webhook
+
+Navigate to the [Database Webhooks settings](https://supabase.com/dashboard/project/_/database/hooks) in your Supabase Dashboard.
+
+1. Enable and create a new hook.
+1. Conditions to fire webhook:
+   1.1. Table: `notifications`
+   1.1. Events: `Insert`
```
Comment (thorwebdev) [inline apps/docs/pages/guides/functions/examples/push-notifications.mdx:105]:
```suggestion
1. Conditions to fire webhook: Select the `notifications` table and tick the `Insert` event. 
```

File: apps/docs/pages/guides/functions/examples/push-notifications.mdx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+1. Webhook configuration: Supabase Edge Functions.
+1. Edge Function:
+   1.1. Method: `POST`
+   1.1. Select which edge function to trigger: `push`
+   1.1. Timeout: `1000`
+1. HTTP Headers:
+   1.1. Content-type: `application/json`
+   1.1. Click "Add new header" > "Add auth header with service key.
+1. Click "Create webhook".
```
Comment (thorwebdev) [inline apps/docs/pages/guides/functions/examples/push-notifications.mdx:114]:
```suggestion
1. Edge Function: Select the `push` edge function and leave the method as `POST` and timeout as `1000`. 
1. HTTP Headers: Click "Add new header" > "Add auth header with service key and leave Content-type: `application/json`.
1. Click "Create webhook".
```

File: examples/user-management/expo-push-notifications/README.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+   1.1. Events: `Insert`
+1. Webhook configuration: Supabase Edge Functions
+1. Edge Function:
+   1.1. Method: `POST`
+   1.1. Select which edge function to trigger: `push`
+   1.1. Timeout: `1000`
+1. HTTP Headers:
+   1.1. Content-type: `application/json`
+   1.1. Click "Add new header" > "Add auth header with service key
+1. Click "Create webhook"
```
Comment (thorwebdev) [inline examples/user-management/expo-push-notifications/README.md:55]:
```suggestion
1. Conditions to fire webhook: Select the `notifications` table and tick the `Insert` event. 
1. Webhook configuration: Supabase Edge Functions.
1. Edge Function: Select the `push` edge function and leave the method as `POST` and timeout as `1000`. 
1. HTTP Headers: Click "Add new header" > "Add auth header with service key and leave Content-type: `application/json`.
1. Click "Create webhook".
```

**Pull request merged**